src/teleop/jetson/jetson_teleop.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+#Node for teleop-related callback functions
+
+from math import copysign
+from enum import Enum
+import rospy as ros
+from sensor_msgs.msg import Joy
+from geometry_msgs.msg import Twist
+from mrover.msg import DriveVelCmd, RAOpenLoopCmd, HandCmd
+
+
+def quadratic(val):
+    return copysign(val**2, val)
+
+def deadzone(magnitude, threshold):
+    temp_mag = abs(magnitude)
+    if temp_mag <= threshold:
+        temp_mag = 0
+    else:
+        temp_mag = (temp_mag - threshold)/(1 - threshold)
+
+    return copysign(temp_mag, magnitude)
+
+
+class Drive:
+    class LogitechAxes(Enum):
+      left_right = 0
+      forward_back = 1
+      twist = 2
+      dampen = 3
+      pan = 4
+      tilt = 5
+
+    #TODO: Reimplement Dampen Switch
+    def teleop_drive_callback(self, msg):
+
+        #teleop_twist_joy message is maxed at 0.5, multiply by 2 to make range [-1,1]
+        linear = msg.linear.x * 2
+        angular = msg.angular.z * 2
+
+        # Convert arcade drive to tank drive
+        # Inversions may need to be done for logitech joystick
+        angular_op = (angular / 2) / (abs(linear) + 0.5)
+        vel_left = linear + angular_op
+        vel_right = linear - angular_op
+
+        # Scale to be within [-1, 1], if necessary
+        if abs(vel_left) > 1 or abs(vel_right) > 1:
+            if abs(vel_left) > abs(vel_right):
+                vel_right /= abs(vel_left)
+                vel_left /= abs(vel_left)
+            else:
+                vel_left /= abs(vel_right)
+                vel_right /= abs(vel_right)
+
+        command = DriveVelCmd()
+        command.left = vel_left
+        command.right = vel_right
+
+        drive_vel_pub = ros.Publisher('/drive_vel_cmd', DriveVelCmd, queue_size=100)
```

Comment:
I noticed that in @niwhsa9 's previous pr, he used Chassis.msg instead of DriveVelCmd. Would Chassis.msg be useful for auton in the sim? If so, you may want to put that code into here. Depending on which controller ESW uses next year, we may even be able to skip the DriveVelCmd and just use Chassis.msg.
NOTE: the only diff between Chassis and DriveVelCmd is that Chassis is in rad/s and is uncapped, while DriveVelCmd is capped between -1 and 1.

See [here](https://github.com/umrover/mrover-ros/blob/amg/drive/src/experimental/rust/drive/src/main.rs) to see what I'm talking about

Reply:
Will default to @niwhsa9 on this one I can change it if need be

Reply:
please go ahead and do the real kinematics math here like I did in the Rust node and output the Chassis.msg with real units. If input desaturation is needed to make L/R wheel velocities feasible, thats fine either in here or ESW but I'd probably lean toward doing it in ESW so its baked into the driver and nobody can accidentally command a value that could be 'damaging'.

Reply:
Yep makes sense, I'm perfectly fine with this